tests/json_infra/conftest.py

```diff
@@ -175,6 +185,29 @@ def pytest_configure(config: Config) -> None:
     config.stash[desired_forks_key] = desired_forks
 
 
+def pytest_collection_modifyitems(config: Config, items: list[Item]) -> None:
+    """Filter test items."""
+    tests_path = config.getoption("tests_path", None)
+    if tests_path is None:
+        return
+
+    with open(tests_path) as f:
+        test_ids = set(x[:-1] for x in f.readlines())
```

Comment:
```suggestion
        test_ids = set(line.strip() for line in f)
```
Otherwise, this could skip the last test in the file, if the file does not end with a new line